locales/en-US/editor.properties

```diff
@@ -208,6 +208,24 @@ PDF_FILE_TITLE=PDF File
 ################
 ## EXTENSIONS ##
 ################
+# extensions/default/InlineBorderRadiusEditor
+ALL_CORNERS=All Corners
```

Comment:
Add a comment above this explaining that "All Corners" is an option that can be selected for border-radius.